test/integration/component/test_ospf.py

```diff
@@ -0,0 +1,477 @@
+
+        # 3) Create ACL Items
+        self.batman_aclitem1 = NetworkACL.create(apiclient=self.apiclient, services={},
+            protocol="ICMP", action="Allow", aclid=self.batman_aclgroup.id, cidrlist=["0.0.0.0/0"])
+        self.assertIsNotNone(self.batman_aclitem1, "Network failed to aclItem")
+        self.logger.debug("Added a network ACL %s to ACL list %s" % (self.batman_aclitem1.id, self.batman_aclgroup.name))
+        
+        self.batman_aclitem2 =  NetworkACL.create( self.apiclient, aclid=self.batman_aclgroup.id, services=self.services["natrule"], 
+                                                   traffictype='Ingress' )
+        
+        self.assertIsNotNone(self.batman_aclitem2, "Network failed to aclItem")
+        self.logger.debug("Added a network ACL %s to ACL list %s" % (self.batman_aclitem2.id, self.batman_aclgroup.name))
+
+        """ create tiers for batman vpc """
+        """
+         32 vpc_tier1_name=batman-tier-1
+         33 vpc_tier1_display_name='"Batman Tier 1"'
+         34 vpc_tier1_gateway=200.100.1.1
+         35 vpc_tier1_netmask=255.255.255.0
+         
+         """
+        self.batman_network_tier1 = Network.create(
+            self.apiclient,
+            self.services["batman_network_tier1"],
+            networkofferingid=self.network_offering.id,
+            zoneid=self.zone.id,
+            aclid=self.batman_aclgroup.id,
+            gateway='200.100.1.1',
+            vpcid=self.batman_vpc.id
+        )
+        self.validate_network(self.batman_network_tier1, self.batman_vpc.id)
+        self.logger.debug("Created batman_network_tier1 with ID: %s" % self.batman_network_tier1.id)
+        
+        
+        """ ----------------------SUPERMAN-------------------------"""
+        # 2) Create ACL
+        self.superman_aclgroup = NetworkACLList.create(apiclient=self.apiclient, services={}, name="acl", description="acl", vpcid=self.superman_vpc.id)
+        self.assertIsNotNone(self.superman_aclgroup, "Failed to create NetworkACL list")
+        self.logger.debug("Created a network ACL list %s" % self.superman_aclgroup.name)
+
+        # 3) Create ACL Item
+        self.superman_aclitem1 = NetworkACL.create(apiclient=self.apiclient, services={},
+            protocol="ICMP", action="Allow", aclid=self.superman_aclgroup.id, cidrlist=["0.0.0.0/0"])
+        self.assertIsNotNone(self.superman_aclitem1, "Network failed to aclItem")
+        self.logger.debug("Added a network ACL %s to ACL list %s" % (self.superman_aclitem1.id, self.superman_aclgroup.name))
+      
+        
+        self.superman_aclitem2 =  NetworkACL.create( self.apiclient, aclid=self.superman_aclgroup.id, services=self.services["natrule"],
+                                        traffictype='Ingress' )
+        self.assertIsNotNone(self.superman_aclitem2, "Network failed to aclItem")
+        self.logger.debug("Added a network ACL %s to ACL list %s" % (self.superman_aclitem2.id, self.superman_aclgroup.name))
```

Comment:
Lines 354-364 appear similar to lines 315-325 -- the previously suggested extraction could be used here as well.